packages/react/src/components/MultiSelect/__tests__/MultiSelect-test.js

```diff
@@ -192,7 +192,9 @@ describe('MultiSelect', () => {
     });
     pressEnter();
 
-    expect(itemNode.getAttribute('data-contained-checkbox-state')).toBe('true');
+    expect(itemNode.getAttribute('data-contained-checkbox-state')).toBe(
+      'false'
+    );
```

Comment:
```suggestion
```

We can probably just remove this. I didn't see that the block below it is the same. We should also rename this test to say `should not toggle selection with enter`